collectors/python.d.plugin/ceph/ceph.chart.py

```diff
@@ -120,6 +120,7 @@ def __init__(self, configuration=None, name=None):
         self.definitions = CHARTS
         self.config_file = self.configuration.get('config_file')
         self.keyring_file = self.configuration.get('keyring_file')
+        self.rados_id = self.configuration.get('rados_id')
```

Comment:
[cleanly](https://github.com/ceph/ceph/blob/bb3a4f227068fc96b8b262057d34892ada63c1c6/src/pybind/rados/rados.pyx#L710)

I like the idea of making it very obvious that the collector is going to attempt to connect as admin, but if someone already has it working and does not update their config file it will still work.

Reply:
ok, please add then and i will approve the PR

Reply:
I'll add it (well once I figure out how to add it to the PR cleanly).

Seems redundant because if you don't specify an ID admin is used, but if this is what you want I'll roll with it.

Reply:
To be honest i said `please add` because you said `I like the idea`. I completely agree that is it redundant 😄